Content.Server/Body/Systems/InternalsSystem.cs

```diff
@@ -38,6 +40,38 @@ public override void Initialize()
         SubscribeLocalEvent<InternalsComponent, ComponentShutdown>(OnInternalsShutdown);
         SubscribeLocalEvent<InternalsComponent, GetVerbsEvent<InteractionVerb>>(OnGetInteractionVerbs);
         SubscribeLocalEvent<InternalsComponent, InternalsDoAfterEvent>(OnDoAfter);
+        SubscribeLocalEvent<BreathToolComponent, ConnectedBreathToolEvent>(OnBreathMaskConnected);
+    }
+
+    public override void Update(float frameTime)
+    {
+        base.Update(frameTime);
+
+        var query = EntityQueryEnumerator<InternalsDelayedActivationComponent>();
+        while (query.MoveNext(out var uid, out var delayComp))
+        {
+            if (_timing.CurTime < delayComp.Time)
+                return;
+
```

Comment:
```suggestion
        var query = EntityQueryEnumerator<InternalsDelayedActivationComponent>();
        while (query.MoveNext(out var uid, out var delayComp))
        {
            if (_timing.CurTime < delayComp.Time)
                continue;

```

return will break for > 1 mask.